src/java/org/apache/cassandra/db/lifecycle/LogReplicaSet.java

```diff
@@ -222,13 +222,28 @@ void printContentsWithAnyErrors(StringBuilder str)
      */
     void append(LogRecord record)
     {
-        Throwable err = Throwables.perform(null, replicas().stream().map(r -> () -> r.append(record)));
+        Throwable err = null;
```

Comment:
@k-rus would it help with testing if we return `failed` from this method?
this way we could check that indeed
a) failed is > 0
b) but it did not throw exception